source/check_config.php

```diff
@@ -133,10 +142,47 @@ function checkVersion( $dom )
     exit;
   }
 }
+
 $lines = file( $conffile );
-$dom->load( $conffile );
+$combinedLines = $lines;
+$lineMappings = array();
+$mappedLineOffset = 0;
+
+foreach( $lines as $line_num => $line ) {
+    if (preg_match('/(\s*)(.*)<include src="([^"]+)"\s*\/>(.*)/', $line, $matches)) {
+        $includedLines = file($resourceDir . $matches[3]);
+        $func = function($value) {
```

Comment:
Anonymous functions were introduced with PHP 5.3 - but the WireGate runs PHP Version 5.2.6-1+lenny16
=> Please change to ordinary function